recipe/meta.yaml

```diff
@@ -11,9 +11,10 @@ source:
   sha256: {{ sha256 }}
   patches:
     - gh3606.patch
+    - gh3895.patch
```

Comment:
This was merged upstream already, so merging it here until we get a new release is fine.